client/state/ui/selectors.js

```diff
@@ -90,3 +91,39 @@ export function isPreviewShowing( state ) {
 export function getPreviewUrl( state ) {
 	return state.ui.currentPreviewUrl;
 }
+
+/**
+ * Returns the current layout focus area
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}  The current layout focus area
+ */
+export function getCurrentLayoutFocus( state ) {
+	// Temporary while we transition from the Flux store
+	return layoutFocusModule.getCurrent();
```

Comment:
I actually tried to do that first, but since it's available and used all kinds of places, we'd have to have access to the dispatcher and `getState` in the module. I couldn't come up with a way to do that. I guess #6669 came up with a way. :-) I may switch over to that, then.